controllers/workloads/console/controller.go

```diff
@@ -592,6 +653,126 @@ func requeueAfterInterval(logger logr.Logger, interval time.Duration) reconcile.
 	return reconcile.Result{Requeue: true, RequeueAfter: interval}
 }
 
+func sessionRecordFileName(ix int) string {
+	return fmt.Sprintf("%s/output%0d", SessionRecVolMount, ix)
+}
+
+func (r *ConsoleReconciler) buildSidecarContainer() corev1.Container {
+	return corev1.Container{
+		Name:            "session-streamer",
+		Image:           r.SessionSidecarImage,
+		ImagePullPolicy: corev1.PullIfNotPresent,
+		VolumeMounts: []corev1.VolumeMount{
+			{
+				Name:      SessionRecVolName,
+				MountPath: SessionRecVolMount,
+				ReadOnly:  false,
+			},
+		},
+		Resources: corev1.ResourceRequirements{
+			Limits: corev1.ResourceList{
+				corev1.ResourceMemory: resource.MustParse("512Mi"),
+				corev1.ResourceCPU:    resource.MustParse("512m"),
+			},
+			Requests: corev1.ResourceList{
+				corev1.ResourceMemory: resource.MustParse("512Mi"),
```

Comment:
These seem pretty beefy, but happy to go with it and tune it down later on if we see that it's excessive

Reply:
These are definitely generous but we're not sure what the actual requirements are yet.